src/Appwrite/Utopia/Database/Validator/Queries/Buckets.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Appwrite\Utopia\Database\Validator\Queries;
+
+use Appwrite\Utopia\Database\Validator\Queries\Base;
+
+class Buckets extends Base
+{
+    public const ALLOWED_ATTRIBUTES = [
+        'enabled',
+        'name',
+        'fileSecurity',
+        'maximumFileSize',
+        'encryption',
+        'antivirus'
+    ];
```

Comment:
This seems error-prone and difficult to maintain, is it always just the indices of the collection? Any way we could map these from `collections.php` config for all the validators?

Reply:
In theory, it could be a boolean value in collections.php alongside each attribute or some kind of `extra` configuration. I will discuss this with Christy. To keep things moving, I am resolving this and making myself a note to address it in the future.